devtools/maven/src/main/java/io/quarkus/maven/DevMojo.java

```diff
@@ -642,13 +643,47 @@ void prepare() throws Exception {
             if (devModeContext.isEnablePreview()) {
                 args.add(DevModeContext.ENABLE_PREVIEW_FLAG);
             }
+
+            propagateUserProperties();
+
             args.add("-jar");
             args.add(tempFile.getAbsolutePath());
             if (argsString != null) {
                 args.addAll(Arrays.asList(CommandLineUtils.translateCommandline(argsString)));
             }
         }
 
+        private void propagateUserProperties() {
+            final String mavenCmdLine = BootstrapMavenOptions.getMavenCmdLine();
```

Comment:
might be minor issue but this will only consider things explicitly set on the command line; it will not honor `.mvn/maven.config`

Reply:
I'd classify it as a separate issue which should be fixed in the bootstrap Maven context/resolver.

Reply:
@maxandersen @aloubyansky we work with values in the `.mvn/maven.config` file. This still results in 

`
Caused by: io.quarkus.bootstrap.resolver.maven.workspace.UnresolvedVersionException: Failed to resolve version '${revision}${sha1}' for com.example:lambda
at io.quarkus.bootstrap.resolver.maven.workspace.UnresolvedVersionException.forGa(UnresolvedVersionException.java:13)
at io.quarkus.bootstrap.resolver.maven.workspace.LocalProject.getVersion(LocalProject.java:195)
at io.quarkus.bootstrap.resolver.maven.workspace.LocalProject.getAppArtifact(LocalProject.java:286)
at io.quarkus.bootstrap.resolver.maven.workspace.LocalProject.getAppArtifact(LocalProject.java:281)
at io.quarkus.bootstrap.BootstrapAppModelFactory.resolveAppModel(BootstrapAppModelFactory.java:275)
... 46 more
`

A workaround is putting the properties in the `pom.xml`. For us that would mean adapting all project to streamline them again.

Reply:
@Thul95 please create a new issue for that. Thanks!